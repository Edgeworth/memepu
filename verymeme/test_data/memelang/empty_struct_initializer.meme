// status: ok
// input: ''
// output: ''
// returns: 0
struct Empty {}
fn main() i8 {
  e := Empty{};
  e : Empty = Empty{};
  ret 0;
}
