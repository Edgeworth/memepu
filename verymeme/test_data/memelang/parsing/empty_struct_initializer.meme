struct Empty {}
fn main() i8 {
  var e: Empty = {};
}
