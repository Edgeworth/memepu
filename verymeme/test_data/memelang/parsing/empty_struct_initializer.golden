File
 Fn
  FnSig
   Typename(main)
   Type(i8)
  StmtBlk
   Var
    VarDecl
     VarRef(e)
     Type(Empty)
    CompoundLit
 Struct
  Typename(Empty)
